test/test_real_pebble.py

```diff
@@ -259,6 +259,33 @@ def stdin_thread():
 
         self.assertEqual(reads, [b'one\n', b'2\n', b'THREE\n'])
 
+    def test_log_forwarding(self):
+        self.client.add_layer("log-forwarder", {
+            "services": {
+                "tired": {
+                    "override": "replace",
+                    "command": "sleep 1",
+                },
+            },
+            "log-targets": {
+                "pretend-loki": {
+                    "type": "loki",
+                    "override": "replace",
+                    "location": "https://example.com",
+                    "services": ["all"],
+                    "labels": {"foo": "bar"},
+                },
+            },
+        }, combine=True)
+        self.client.replan_services()
```

Comment:
There's actually no need for the `replan_services` in this test, as we're just testing the plan itself, not that any services were (re)started.

Reply:
Thanks for adding the test!